_includes/breadcrumbs.html

```diff
@@ -1,12 +1,19 @@
 ﻿{% assign parent_dirs=page.url | replace_first: '/', '' | split: '/' %}
 {% assign link='/' %}
-<ol class="breadcrumb">
+<ol class="breadcrumb" vocab="http://schema.org" typeof="BreadcrumbList">
+    {% assign position = 1 %}
     {% for dir in parent_dirs %}
         {% if forloop.last %}
-            <li class="active">{{ dir }}</li>
```

Comment:
It looks like I partially reimplemented the classes but failed to carry it through -- I'll open a PR to fix this soon, or you can revert the merge of this PR and I can append it to this one.